.github/workflows/release.yml

```diff
@@ -22,6 +22,13 @@ jobs:
     runs-on: ubuntu-latest
     name: "Release a new version"
     steps:
+        - id: killswitch
+          if: ${{ !github.event.inputs.force }}
+          run: |
+            if curl -s "https://api.github.com/repos/$GITHUB_REPOSITORY/issues?state=open&labels=release-blocker" | grep -Pzvo '\[[\s\n\r]*\]'; then
+              echo "Open release-blocking issues found, cancelling release...";
+              curl -sf -X POST -H 'Accept: application/vnd.github.v3+json' -H 'Authorization: token ${{ secrets.GITHUB_TOKEN }}' https://api.github.com/repos/$GITHUB_REPOSITORY/actions/runs/${{ github.run_id }}/cancel;
```

Comment:
Isn't it enough to just exit with an error instead of calling the API?

Reply:
Unfortunately not: https://github.community/t/is-there-a-way-to-cancel-a-running-workflow-within-one-of-its-jobs/17493/3